### PR TITLE
[TACHYON-1020] Added JavaDoc for InodeLastModificationTimeEntry

### PR DESCRIPTION
**Pull request by sidharthgoel**
https://tachyon.atlassian.net/browse/TACHYON-1020

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

**Comment by gpang** [comment]
Thanks for the patch! Could you also modify the PR description to add a link back to the jira ticket?

**Comment by sidharthgoel** [comment]
@gpang I accidentally pushed my new change into this existing pull request...do I need to reset something and create a new pull request with my change ("Can set number of zookeeper servers")?

File: servers/src/main/java/tachyon/master/file/journal/InodeLastModificationTimeEntry.java [hunk 1]
```diff
@@ -22,10 +22,18 @@
...
 import tachyon.master.journal.JournalEntryType;
 
+/**
+ * This class represents a journal entry for the last time an inode was modified.
+ */
 public final class InodeLastModificationTimeEntry implements JournalEntry {
   protected final long mId;
   protected final long mLastModificationTimeMs;
 
+    /**
```
Comment (apc999) [inline servers/src/main/java/tachyon/master/file/journal/InodeLastModificationTimeEntry.java]: indentation seems off from line 32 to line 36

**Comment by calvinjia** [comment]
@sidharthgoel thanks for the patch, could you clean up the commit history (it seems like you committed some unrelated change)?

**Comment by ooq** [comment]
jenkins, test this please

**Comment by ooq** [comment]
@sidharthgoel your patch probably fails the style checker.

**Comment by sidharthgoel** [comment]
jenkins, test this please

**Comment by sidharthgoel** [comment]
@calvinjia @gpang can you help me understand what's going wrong here - the only diff is the JavaDoc, and the style is fixed, so could it be a jenkins issue?

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
LGTM

**Pull request merged**